src/Stream.java

```diff
@@ -5,21 +5,30 @@
 public class Stream {
     public static void main(String[] args) {
 
-        IntStream numbersStream = IntStream.iterate(0, x -> x + 1);
+        IntStream numbersStream = IntStream.iterate(1, x -> x + 1);
```

Comment:
This list filling with integers seems weird  but lets leave it if it works.

Reply:
yes, it working, i had to shift it by "1" for aesthetic value when it's printed in console.